db/migrate/20180814154354_create_mitigation_actions.rb

```diff
@@ -0,0 +1,21 @@
+class CreateMitigationActions < ActiveRecord::Migration[5.1]
+  def change
+    create_table :mitigation_actions do |t|
+      t.integer :mitigation_category_id, optional: false
+      t.string :name, optional: false
```

Comment:
I'd go for `text` to avoid unpleasant surprises

Reply:
Done